script.js

```diff
@@ -0,0 +1,68 @@
+
+let form = document.querySelector('form');
+
+let search_text = "";
+const coursesExplorerTitle = document.querySelector('.courses-explore-header-title');
+const coursesExplorerDesc = document.querySelector('.courses-explore-header-par');
+const coursesList = document.querySelector('.courses-list');
+const courseTemplate = document.querySelector("#courses-list");
+
+let courses = [];
+
+
+/// display all courses
+fetch("http://localhost:3000/python")
+  .then(res => res.json())
+  .then((json) => {
+    let data = json.courses;
+    coursesExplorerTitle.innerText = json.header;
+    coursesExplorerDesc.innerText = json.description;
+
+    courses = data.map((data) => {

```

Comment:
I think you can use `forEach` here instead of `map`, we usually use map when you need to map the array elements to a new array